src/common/common-md-flag/common-md-flag.vcxproj

```diff
@@ -51,15 +51,16 @@
   </PropertyGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
     <ClCompile>
-      <PrecompiledHeader>Use</PrecompiledHeader>
+      <PrecompiledHeader>NotUsing</PrecompiledHeader>

```

Comment:
any reason for this change?

Reply:
This was changed so the `pch.h` file from the `common` project is used instead of the one that used to be inside of the `common-md-flag` project.